### PR TITLE
139294153 update erb view rendering

### PR DESCRIPTION
**Pull request by drod1000**
@jdconrad89 @Laszlo-JFLMTCO @mollybrown 
This PR moves the rendering of react components to the views by redirecting to the appropriate action. I added a few tests to cover the lines of code that were untested.

I went ahead and added simple cov to the spec helper on this branch as well.

@neight-allen

### CODE REVIEW

**Comment by neight-allen** [comment]
:+1: This is definitely preferable.

**Comment by jdconrad89** [comment]
@drod1000 I agree with Nate this is much better! Looks great! Thanks for adding SimpleCov as well!

File: spec/features/portfolios/user_can_edit_portfolio_spec.rb [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
+    portfolio = @user.create_portfolio(linkedin_url: "linkedin.com", full_name: "Anthony Ciccone", github_url: "github.com", email: "test@test.com", title: "software developer", bio: "about myself here")
+    visit dashboard_path
+  end
+
   context 'as a new logged in user' do
     it "can edit an existing portfolio" do
-      user = new_user
-      portfolio = user.create_portfolio(linkedin_url: "linkedin.com", full_name: "Anthony Ciccone", github_url: "github.com", email: "test@test.com", title: "software developer", bio: "about myself here")
-      visit dashboard_path
-
```
Comment (Laszlo-JFLMTCO) [inline spec/features/portfolios/user_can_edit_portfolio_spec.rb:9]: @drod1000 Much cleaner now!!!

File: app/controllers/users/portfolios_controller.rb [hunk 2]
```diff
@@ -39,7 +39,7 @@ def update
 
       render js: "/dashboard"
     else
-      render component: 'Portfolio', props: { user: current_user, portfolio: current_user.portfolio, slug: current_user.slug, avatar: current_user.portfolio.avatar.url(:regular), locations: Location.distinct_city_states, userLocations: current_user.portfolio.locations }, status: 400
+      redirect_to edit_user_portfolio_path(current_user), status: 400
```
Comment (Laszlo-JFLMTCO) [inline app/controllers/users/portfolios_controller.rb:42]: @drod1000 Wow, great job, this is much-much cleaner now!

File: app/controllers/users/portfolios_controller.rb [hunk 3]
```diff
@@ -20,7 +20,7 @@ def create
 
       render js: "/dashboard"
     else
-      render component: 'Portfolio', props: { user: current_user, portfolio: current_user.portfolio, slug: current_user.slug, locations: Location.distinct_city_states}, status: 400
+      redirect_to new_user_portfolio_path(current_user), status: 400
```
Comment (Laszlo-JFLMTCO) [inline app/controllers/users/portfolios_controller.rb:23]: @drod1000 Wow, great job, this is much-much cleaner now!

**Comment by Laszlo-JFLMTCO** [comment]
@drod1000 Thank you for the cleanup effort! Looks much better! I am ok to merge.

**Pull request merged**